src/extensions/extensionInterface.ts

```diff
@@ -196,10 +196,18 @@ export interface Plugins {
     sortingUnitMetrics: {[key: string]: SortingUnitMetricPlugin}
 }
 
-export interface SortingViewProps {
+interface ViewProps {
```

Comment:
These are the props that SortingViewProps, RecordingViewProps, and SortingUnitViewProps all have in common.